### ISSUE TITLE
Accessibility issue: date dividers are not marked as header

### ISSUE THREAD

**Issue by mennovogel** [parley-messaging/android-library#92]
From our accessibility audit we got the following issue:

_The heading text is not marked up as a heading. Simply making text bold or larger does not communicate its role to assistive technologies. Ensure headings are marked up properly so assistive technologies can identify them and convey the content structure accurately.

The dates are headings._

![Image](https://github.com/user-attachments/assets/2e1b8e08-297f-44db-863b-2c2e7563e0ff)

The dates need to be marked as heading for screenreader.

**Comment by mennovogel** [comment]
I've created https://github.com/parley-messaging/android-library/pull/95 to fix this.

**Comment by alexkok** [comment]
Fixed in [3.11.3](https://github.com/parley-messaging/android-library/releases/tag/3.11.3) 🚀

**Comment by LauBCra** [comment]
Unfortunately, this is only partially fixed. The date can be selected using the "heading" setting of the screenreader, but the screenreader doesn't read out that the date is a heading. Expected would be "<date>, heading". This gives the user context about what is one the screen.

Also it's not possible for the screenreader user to swipe between date headers that are not in view, I suspect this has something to do with the way the message paging is done. This makes is not very usable, as the user still needs to swipe through all the messages and can't use the heading setting, to easily skip back through the messages..
